src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/JndiUserDirectoryDAO.scala

```diff
@@ -20,6 +20,10 @@ class JndiUserDirectoryDAO(providerUrl: String, user: String, password: String,
     ctx.bind(p.name, p)
   }
 
+  override def removeUser(user: RawlsUser): Future[Unit] = withContext { ctx =>
+    ctx.unbind(new Person(user).name)
```

Comment:
I'm not entirely sure but I think that you may also want to disable the user to remove their LDAP entry from the authorized group (it's possible that by removing a user this entry is already removed, I don't know). Since users are enabled by default now it probably won't matter but it'll make LDAP cleaner.